tests/ui/check-cfg/cfg-value-for-cfg-name.rs

```diff
@@ -0,0 +1,13 @@
+// #120427
+// This test checks that when a single cfg has a value for user's specified name
+// suggest to use `#[cfg(target_os = "linux")]` instead of `#[cfg(linux)]`
+//
+// check-pass
+// compile-flags: -Z unstable-options
+// compile-flags: --check-cfg=cfg(feature,values())
```

Comment:
No need to specify the `feature` cfg; it's irrelevant to the test since we are only interested well known names and values.
```suggestion
// compile-flags: --check-cfg=cfg()
```